app/javascript/components/region-form/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import createSchema from './region-form.schema';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import { API } from '../../http_api';
+import miqRedirectBack from "../../helpers/miq-redirect-back";
+
+const RegionForm = ({ id, maxDescLen }) => {
+  const [initialValues, setInitialValues] = useState({});
+  const [isLoading, setIsLoading] = useState(!!id);
```

Comment:
You don't really need this second one, you can just test against the `initialValues` in the rendering.

Reply:
Also if you can always use a single state object like this:
```
const [{ description, isLoading }, setState] = useState({});
```
But in this case you don't really need it.